inc/views/nav_walker.php

```diff
@@ -216,6 +218,61 @@ public function enqueue_mega_menu_style() {
 		wp_style_add_data( 'neve-mega-menu', 'rtl', 'replace' );
 		wp_style_add_data( 'neve-mega-menu', 'suffix', '.min' );
 		wp_enqueue_style( 'neve-mega-menu' );
+
+		$style = <<<'CSS'
```

Comment:
If this style will apply only with the new skin, I think we can probably condition it as such. If it doesn't affect anything else, we can just add it inside the mega-menu SCSS file instead of inline. 🚀

Reply:
I've conditioned the style to the new skin only since it hooks into the `.left`, `.center`, `.right` classes